runtime-modules/content/src/benchmarks/benchmarking.rs

```diff
@@ -2249,147 +2261,1096 @@ benchmarks! {
+    //   - `price - royalty` is non-zero
+    // INPUT COMPLEXITY
+    make_english_auction_bid {
+        let (
+            video_id,
+            (curator_account_id, actor, channel_id, _)
+        ) = setup_worst_case_scenario_mutable_video::<T>(
+            Some(T::MaxNumberOfAssetsPerVideo::get()),
+            T::StorageBucketsPerBagValueConstraint::get().max() as u32,
+        )?;
+
+        let ((nft_owner_actor, owner_account), participant_id, participant_account_id) = setup_nft_in_english_auction::<T>(
+            curator_account_id,
+            actor,
+            video_id,
+            true,
+        )?;
+
+        set_all_channel_paused_features::<T>(channel_id);
+
+        fastforward_by_blocks::<T>(2u32.into());
+        let balance_pre = Balances::<T>::usable_balance(participant_account_id.clone());
+        let _ = add_english_auction_bid::<T>(participant_account_id.clone(), participant_id, video_id);
+        let buy_now_amount = BUY_NOW_PRICE.into();
+
+        let origin = RawOrigin::Signed(participant_account_id.clone());
+    }: _(origin, participant_id, video_id, buy_now_amount)
+        verify {
+            assert!(matches!(Pallet::<T>::video_by_id(video_id).nft_status, Some(Nft::<T> {
+                transactional_status: TransactionalStatus::<T>::Idle,
+                ..
+            })));
+            assert_eq!(
+                Balances::<T>::usable_balance(participant_account_id),
+                balance_pre - buy_now_amount,
+            )
+        }
+
+    // WORST CASE SCENARIO:
+    // - curator owned channel
+    // - curator number is max
+    // - curator has max number of agent permissions
+    // - channel has max size:
+    //   - all feature paused (except necessary ones for extr to succeed)
+    //   - max channel assets
+    //   - max collaborators
+    // - video has max size
+    //   - max video assets
+    // - nft limits are set
+    // - complete payment has max complexity:
+    //   - nft owner is a member (different from channel owner)
+    //   - royalty is non-zero
+    //   - `price - royalty` is non-zero
+    // INPUT COMPLEXITY
+    settle_english_auction {
+        let (
+            video_id,
+            (curator_account_id, actor, channel_id, _)
+        ) = setup_worst_case_scenario_mutable_video::<T>(
+            Some(T::MaxNumberOfAssetsPerVideo::get()),
+            T::StorageBucketsPerBagValueConstraint::get().max() as u32,
+        )?;
+
+        let ((nft_owner_actor, owner_account), participant_id, participant_account_id) = setup_nft_in_english_auction::<T>(
+            curator_account_id.clone(),
+            actor,
+            video_id,
+            true,
+        )?;
+
+        set_all_channel_paused_features::<T>(channel_id);
+        let origin = RawOrigin::Signed(owner_account.clone());
+
+        fastforward_by_blocks::<T>(2u32.into());
+        let _ = add_english_auction_bid::<T>(participant_account_id.clone(), participant_id, video_id);
```

Comment:
The bid is made for `min_starting_price` which is currently set to `1`. This prevents the worst case scenario from being triggered

Reply:
I set that value to `nft_buy_now_price - 1`.
The `min_starting_price` is `10` hovewer...

Reply:
In `node/src/chain_spec/content_config.rs` it's `1` and that's the one being used when weights are generated I think